data-table/__tests__/table.js

```diff
@@ -20,6 +20,29 @@ describe('basic tests to ensure createDataTable can function well', () => {
         expect(contents.substr(0, 12)).toEqual("+ Add column");
     });
 
+    test("test default row name", () => {
+        table.createDataTable({
+            'wrapperDivId': 'div-id',
+            'rowsName': 'div-row',
+            'columnsName': 'div-col'
+        });
+        const contents = document.getElementsByClassName('dt_cell')[1].textContent;
```

Comment:
Using [1] is a bit of a magic number here - can you explain in a comment why you're using [1] instead of [0]?